docs/DeveloperGuide.md

```diff
@@ -154,6 +154,60 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Find feature
+
+#### What is the feature about?
+The `find` mechanism is facilitated by `AddressBook3`. The `find` command in `TAlent Assistant™`
```

Comment:
Hey @charmainehly, I think this should be `AddressBook`. If I'm not mistaken, it's referencing to the `AddressBook` class.

Reply:
Got it! Let me fix this in a future commit and PR. Thanks for catching this :)